packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -132,7 +132,7 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call
       [result sendNotImplemented];
     }
   } else if ([@"create" isEqualToString:call.method]) {
-    FLTRequestCameraPermissionWithCompletionHandler(^(FlutterError *error) {
+    FLTRequestCameraPermission(/*forAudio*/ false, ^(FlutterError *error) {
```

Comment:
Instead of this BOOL (which should be `NO` not `false` btw), how about hiding the details of `forAudio` and creating `FLTRequestCameraPermission` and `FLTRequestAudioPermission` functions that can share unexposed logic?

Reply:
can do